### ISSUE TITLE
Add example showing how to plot a vector layer over a raster data with matplotlib

### ISSUE THREAD

**Issue by Robinlovelace** [geopython/geopython-workshop#111]
As discussed with demonstrators at FOSS4G, this could be a good addition. Reproducible example: https://github.com/matplotlib/matplotlib/issues/23704#issuecomment-1222391297

**Comment by justb4** [comment]
@Robinlovelace agree, good addition. Timing though is tight for this workshop. We are moving parts of material to "advanced" thus optional parts, to the end of sections. We are busy editing for FOSS4G 2023. Is this something you could contribute in a PR?
